tests/resources/modules/d8/woot/woot.services.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  woot.commands:
+    - woot.manager
+    - woot.command
+    - greet.command
+services:
+  woot.manager:
+    class: Drupal\woot\WootManager
+    arguments: ['@current_user']
+  woot.command:
+    class: Drupal\woot\Command\WootCommands
```

Comment:
IMO service name and classname for woot.command should both be singular or both be plural